pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/RuntimeASTTransformer.groovy

```diff
@@ -181,7 +302,7 @@ class RuntimeASTTransformer {
      * @param valueType Either {@link ModelASTInternalFunctionCall} for credentials or {@link ModelASTValue} for env.
      * @return The AST for instantiating the resolver.
      */
-    private Expression generateEnvironmentResolver(@Nonnull Map<ModelASTKey, ModelASTEnvironmentValue> variables, @Nonnull Class valueType) {
+    private Expression generateEnvironmentResolver(@Nonnull Map<ModelASTKey, ModelASTEnvironmentValue> variables, @Nonnull Class valueType, boolean disableWrapping = false) {
```

Comment:
SpotBugs doesn't like that when `RuntimeASTTransformer` is converted to Java, a version of this method without `disableWrapping` is generated (that's how default parameters are implemented by Groovy), which is never called:

> Private method org.jenkinsci.plugins.pipeline.modeldefinition.parser.RuntimeASTTransformer.generateEnvironmentResolver(Map, Class) is never called [org.jenkinsci.plugins.pipeline.modeldefinition.parser.RuntimeASTTransformer] In RuntimeASTTransformer.groovy UPM_UNCALLED_PRIVATE_METHOD

If `disableWrapping` is always specified, and there's not concern about backwards compat, then you can just drop the default expression `= false`, otherwise just suppress the warning (or convert the class to Java since it's in `src/main` anyway, but that's more work).